Assignment1/pom.xml

```diff
@@ -57,9 +57,29 @@
 	    <version>2.11.0</version>
 	</dependency>
   
+	<!-- https://mvnrepository.com/artifact/com.fasterxml.jackson.core/jackson-databind -->
```

Comment:
Nice work on making sure that this is a Maven project and adding the config to the repository.